components/destination/AddWishlistItem.js

```diff
@@ -3,37 +3,51 @@
 import { useState } from 'react';
 import styles from './destination.module.css';
 
-export  const AddWishlistItem = ({
-  onAddWishlistItem,
-}) => {
-  const [thumbnail, onThumbnailChange] = useState('/destination/image-europa.png');
-  // TASK - React 1 week 3
-  // 1. Add a useState for the handling the <input id="customWishlist" type="text" />
-  // 2. Connect the onThumbnailChange to the <select>
+export const AddWishlistItem = ({ onAddWishlistItem }) => {
+  const [customWishlist, setCustomWishlist] = useState('');
+  const [thumbnail, setThumbnail] = useState('/destination/image-europa.png');
+
+  const handleInputChange = (e) => {
+    setCustomWishlist(e.target.value);
+  
```

Comment:
Let's get into the habit of formatting the code before committing it. You can use an extension like Prettier for VS Code.

Reply:
ok, sorry forgot to do that.